cate/webapi/rest.py

```diff
@@ -487,6 +493,159 @@ def get(self, base_dir, res_id):
             self.finish()
 
 
+def save_files(files: Sequence, target_dir: str):
+    for file in files:
+        file_path = os.path.join(target_dir, file.filename)
+        if not os.path.exists(target_dir):
+            os.mkdir(target_dir)
+
+        with open(file_path, 'wb') as fp:
+            fp.write(file.body)
+
+    return 'Files: ' + ', '.join([file.filename for file in files])
+
+
+def ensure_str(value: Any) -> str:
```

Comment:
I guess, this function should be private:

```suggestion
def _ensure_str(value: Any) -> str:
```

Otherwise move it to cate.util and provide meaningful name.